app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id])
+    new_comment = current_user.comments.new(post_id: @post.id,
+                                            author_id: current_user.id, text: comment_text)
```

Comment:
- [ ] Using this logic, you will always get errors that author_id is not an attribute for the Comment model. I would advise you to change this logic to 👍 😎:

```ruby
@post = Post.find(params[:post_id])
 new_comment = Comment.create(text: comment_text[:text], user_id: current_user.id, post_id: @post.id)
```